### ISSUE TITLE
Problems locking KeePass session with KeeAgent installed

### ISSUE THREAD

**Issue by phinor** [dlech/KeeAgent#182]
Having installed the KeeAgent 0.8.1 into KeePass 2.34, I can no longer lock my KeePass database using the "Ctrl + L" shortcut.

Steps to reproduce:

- Install KeeAgent 0.8.1 on KeePass 2.34
- Load at least one certificate when the database loads (either to KeeAgent Server or Pageant)
- Open Database
- Press Ctrl+L
- Exception is generated, workspace doesn't lock
- Click "Continue"
- Click "File > Lock Workspace"
- Workspace locks
- Unlock workspace
- Press Ctrl + L
- Workspace locks
- Unlock workspace
- Bring another window to foreground
- Return to KeePass
- Press Ctrl + L
- Exception is generated, workspace doesn't lock

If the KeeAgent extension is removed, then no exception is generated and the workstation locks as expected.

The following Exception is generated:

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at KeeAgent.KeeAgentExt.PwEntryContextMenuLoadKeyItem_Clicked(Object sender, EventArgs e)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.ProcessCmdKey(Message& m, Keys keyData)
   at System.Windows.Forms.ToolStripManager.ProcessShortcut(Message& m, Keys shortcut)
   at System.Windows.Forms.Form.ProcessCmdKey(Message& msg, Keys keyData)
   at KeePass.Forms.MainForm.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.Control.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.ContainerControl.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.Control.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.Control.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.ContainerControl.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.Control.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.Control.ProcessCmdKey(Message& msg, Keys keyData)
   at System.Windows.Forms.Control.PreProcessMessage(Message& msg)
   at System.Windows.Forms.Control.PreProcessControlMessageInternal(Control target, Message& msg)
   at System.Windows.Forms.Application.ThreadContext.PreTranslateMessage(MSG& msg)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
KeePass
    Assembly Version: 2.34.0.15991
    Win32 Version: 2.34.0.0
    CodeBase: file:///C:/Program%20Files%20(x86)/KeePass%20Password%20Safe%202/KeePass.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
WordSequence
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/Program%20Files%20(x86)/KeePass%20Password%20Safe%202/WordSequence.DLL
----------------------------------------
KeeAgent
    Assembly Version: 0.8.1.0
    Win32 Version: 0.8.1.0
    CodeBase: file:///C:/Users/Philip/AppData/Local/KeePass/PluginCache/rTaC6uNypLQpue8kaYl4/KeeAgent.dll
----------------------------------------
SshAgentLib
    Assembly Version: 0.8.1.0
    Win32 Version: 0.8.1.0
    CodeBase: file:///C:/Users/Philip/AppData/Local/KeePass/PluginCache/rTaC6uNypLQpue8kaYl4/SshAgentLib.DLL
----------------------------------------
SshAgentLib.WinForms
    Assembly Version: 0.8.1.0
    Win32 Version: 0.8.1.0
    CodeBase: file:///C:/Users/Philip/AppData/Local/KeePass/PluginCache/rTaC6uNypLQpue8kaYl4/SshAgentLib.WinForms.DLL
----------------------------------------
KeePassFaviconDownloader
    Assembly Version: 2.0.9.27338
    Win32 Version: 1.9.0.0
    CodeBase: file:///C:/Users/Philip/AppData/Local/KeePass/PluginCache/Lc6pDfFVhS74cWF5srFB/KeePassFaviconDownloader.dll
----------------------------------------
QrCodeGenerator
    Assembly Version: 2.0.12.0
    Win32 Version: 2.0.12.0
    CodeBase: file:///C:/Users/Philip/AppData/Local/KeePass/PluginCache/G2dmR2a1Ja3qhSZuLVPf/QrCodeGenerator.dll
----------------------------------------
KeePass.XmlSerializers
    Assembly Version: 2.34.0.15991
    Win32 Version: 2.34.0.15991
    CodeBase: file:///C:/Program%20Files%20(x86)/KeePass%20Password%20Safe%202/KeePass.XmlSerializers.DLL
----------------------------------------
System.Security
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Security/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Security.dll
----------------------------------------
Microsoft.GeneratedCode
    Assembly Version: 1.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
BouncyCastle.Crypto
    Assembly Version: 1.7.4137.9688
    Win32 Version: 1.7.4137.9688
    CodeBase: file:///C:/Users/Philip/AppData/Local/KeePass/PluginCache/rTaC6uNypLQpue8kaYl4/BouncyCastle.Crypto.DLL
----------------------------------------
Chaos.NaCl
    Assembly Version: 0.1.0.0
    Win32 Version: 0.1.0.0
    CodeBase: file:///C:/Users/Philip/AppData/Local/KeePass/PluginCache/rTaC6uNypLQpue8kaYl4/Chaos.NaCl.DLL
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1586.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.



```

**Comment by dlech** [comment]
Hmm... I thought we fixed this already. We have [conflicting shortcut keys](https://github.com/dlech/KeeAgent/blob/70b16d20026747198c63ea75539915d713d61b0a/KeeAgent/KeeAgentExt.cs#L357) (KeeAgent is trying to use Ctrl+L to load a key for the selected password entry).

**Comment by dlech** [comment]
Ah, in #138, we changed it from Ctrl+K to Ctrl+L. Any suggestion for a new shortcut?

**Comment by dlech** [comment]
As far as I can tell, there are very few letters that are not already used. with Ctrl: D, M, T, Y.

So, we can risk a future collision or we can try a combination of more keys.

**Comment by dlech** [comment]
Fix has been released in [KeeAgent 0.9.0](http://lechnology.com/2017/01/keeagent-beta-0-9-0-released/). Please test.
